hashids_test.go

```diff
@@ -73,7 +73,7 @@ func TestEncodeDecodeEpoch(t *testing.T) {
 
 func TestEncodeDecodeHex(t *testing.T) {
 	hdata := NewData()
-	hdata.MinLength = 30
+	hdata.MinLength = 8
```

Comment:
Why do you change the `MinLength`? What is the result if you leave it at 30?

Reply:
thanks for replying. i had returned it to **'MinLength = 30'**
and you can check it with js example in: https://codepen.io/anon/pen/arVLpG